testsuite/test_nocasedict.py

```diff
@@ -374,23 +369,19 @@ def test_NocaseDict_init(
 @pytest.mark.parametrize(
     "desc, kwargs, exp_exc_types, exp_warn_types, condition",
     TESTCASES_NOCASEDICT_GETITEM)
-@pytest_extensions.test_function
-def test_NocaseDict_getitem(
-        desc, kwargs, exp_exc_types, exp_warn_types, condition):
+@pytest_extensions.test_function_new
+def test_NocaseDict_getitem(testcase,
+                            obj, key, exp_value):
     """
     Test function for NocaseDict.__getitem__() / ncd[key]
     """
 
-    obj = kwargs['obj']
-    key = kwargs['key']
-    exp_value = kwargs['exp_value']
-
     # The code to be tested
     act_value = obj[key]
 
     # Verify that an exception raised in this function is not mistaken
     # to be the expected exception
-    assert exp_exc_types is None
+    assert testcase.exp_exc_types is None
```

Comment:
comment on line above.  'mistaken as the expected exception.

Reply:
DONE, plus changed verify to ensure